### PR TITLE
Authorization module is working

### PR DESCRIPTION
**Pull request by Alexey007gold**

### CODE REVIEW

File: services/authorization/authorization-api/authorization-api.iml [hunk 1]
```diff
@@ -1,18 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (Dominys) [inline services/authorization/authorization-api/authorization-api.iml]: @Alexey007gold remove IDEA files from git

File: services/authorization/authorization-api/src/main/java/com/softgroup/authorization/api/message/LoginRequest.java [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 import com.softgroup.common.protocol.RequestData;
+import org.springframework.stereotype.Component;
 
 /**
  * @author odin
  * @since 20.02.17.
  */
+
+@Component
 public class LoginRequest implements RequestData {
```
Comment (Dominys) [inline services/authorization/authorization-api/src/main/java/com/softgroup/authorization/api/message/LoginRequest.java:12]: @Alexey007gold why component?

File: services/authorization/authorization-api/src/main/java/com/softgroup/authorization/api/message/RegisterRequest.java [hunk 3]
```diff
@@ -1,11 +1,44 @@
...
+import org.springframework.stereotype.Component;
 
 /**
  * @author odin
  * @since 20.02.17.
  */
-public class RegisterRequest implements RequestData{
+
+@Component
+public class RegisterRequest implements RequestData {
```
Comment (Dominys) [inline services/authorization/authorization-api/src/main/java/com/softgroup/authorization/api/message/RegisterRequest.java:12]: @Alexey007gold why component?

File: services/authorization/authorization-api/src/main/java/com/softgroup/authorization/api/message/SmsConfirmRequest.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+import com.softgroup.common.protocol.RequestData;
+import org.springframework.stereotype.Component;
+
+/**
+ * Created by alex on 24.02.17.
+ */
+
+@Component
+public class SmsConfirmRequest implements RequestData {
```
Comment (Dominys) [inline services/authorization/authorization-api/src/main/java/com/softgroup/authorization/api/message/SmsConfirmRequest.java]: @Alexey007gold why component?

File: services/authorization/authorization-impl/authorization-impl.iml [hunk 5]
```diff
@@ -1,18 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (Dominys) [inline services/authorization/authorization-impl/authorization-impl.iml]: @Alexey007gold remove IDEA files

**Comment by Dominys** [comment]
@Alexey007gold remove all database related changes.

**Comment by Dominys** [comment]
@Alexey007gold do not merge master in to branch! It's making mess in commit history. Proper way to update branch is rebase.

**Comment by Alexey007gold** [comment]
Don't now how did it happen))

**Pull request merged**